.github/workflows/checks.yml

```diff
@@ -22,7 +22,20 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
-      - uses: actions/setup-node@v3
+      - uses: actions/setup-node@v4
       - uses: ./.github/actions/security/lockfile
         with:
           allowedHosts: 'yarn'
+
+  install-integrity:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 20.x
+      - name: Monorepo install
```

Comment:
While `yarn dedupe` does not "require" a prior install (it will do it for you), good tip is to pre-install to keep the benefit of yarn-nm-install (more than 2x faster thx to caching)